blazegraph_load_test.sh

```diff
@@ -10,7 +10,69 @@ if [ -z "$BG_SPARQL" ]; then
 fi
 
 # ---
+# Create or recreate the test namespace
+
+# parse the namespace from the BG_SPARQL URI
+export bg_url=$(echo $BG_SPARQL | sed -e 's#\(.*\)/namespace.*#\1#')
+export bg_namespace=$(echo $BG_SPARQL | sed -e 's#.*namespace/\(.*\)/sparql#\1#')
```

Comment:
Good observations, just wondering if they are required changes to merge this PR or suggestions?  I happen to like it as it is and I think these suggestions make for additional work that doesn't really change any functionality.

1. Yes, perhaps that's true but it doesn't really make much difference to the rest of the script and I find it clearer this way.  Essentially, this is the root of all the URIs for the blazegraph app.

2. I find it easier to read `sed` patterns because they tell me something about where and how the string is modified, whereas `cut` doesn't tell me anything about it unless I translate the numbers into something (for me, that's a greater cognitive load than looking at a regex pattern, perhaps because I look at some kind of regex almost every day).